yt_dlp/extractor/tencent.py

```diff
@@ -47,6 +58,11 @@ def _get_video_api_response(self, video_url, video_id, series_id, subtitle_forma
             'sphttps': '1',  # Enable HTTPS
             'otype': 'json',
             'spwm': '1',
+            'hevclv': '28',  # Enable HEVC
```

Comment:
I thought this meant we are "enabling" hevc like the comment says, not extracting ONLY hevc. We want to extract the old avc formats too. Would you be able to fix it, @Hill-98